node/src/server-modules/GlideJson.ts

```diff
@@ -381,23 +386,132 @@ export class GlideJson {
+        >
+    > {
         const args = ["JSON.RESP", key];
 
         if (options) {
             args.push(options.path);
         }
 
-        return _executeCommand<ReturnTypeJson<GlideString>>(client, args);
+        return _executeCommand<
+            UniversalReturnTypeJson<
+                (number | GlideString) | (number | GlideString | null) | null
+            >
+        >(client, args);
+    }
+
+    /**
+     * Returns the length of the JSON string value stored at the specified `path` within
+     * the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) path - The path within the JSON document, Defaults to root if not provided.
+     * @returns ReturnTypeJson:
+     *     - For JSONPath (path starts with `$`):
+     *       - Returns a list of integer replies for every possible path, indicating the length of
+     *         the JSON string value, or <code>null</code> for JSON values matching the path that
+     *         are not string.
+     *     - For legacy path (path doesn't start with `$`):
+     *       - Returns the length of the JSON value at `path` or `null` if `key` doesn't exist.
+     *       - If multiple paths match, the length of the first matched string is returned.
+     *       - If the JSON value at`path` is not a string or if `path` doesn't exist, an error is raised.
+     *     - If `key` doesn't exist, `null` is returned.
+     *
+     * @example
+     * ```typescript
+     * console.log(await GlideJson.set(client, "doc", "$", '{a:"foo", nested: {a: "hello"}, nested2: {a: 31}}"));
+     * // Output: 'OK' - Indicates successful setting of the value at path '$' in the key stored at `doc`.
+     * console.log(await GlideJson.strlen(client, "doc", {path: "$..a"}));
+     * // Output: [3, 5, null]; - The length of the string values at path '$..a' in the key stored at `doc`.
+     *
+     * console.log(await GlideJson.strlen(client, "doc", {path: "nested.a"}));
+     * // Output: 5; - The length of the JSON value at path 'nested.a' in the key stored at `doc`.
+     *
+     * console.log(await GlideJson.strlen(client, "doc", {path: "$"}));
+     * // Output: [null] - Returns an array with null since the value at root path does in the JSON document stored at `doc` is not a string.
+     *
+     * console.log(await GlideJson.strlen(client, "non_existent_key", {path: "."}));
+     * // Output: null - return null if key does not exist.
+     * ```
+     */
+    static async strlen(
+        client: BaseClient,
+        key: GlideString,
+        options?: { path: GlideString },
+    ): Promise<ReturnTypeJson<number>> {
+        const args = ["JSON.STRLEN", key];
+
+        if (options) {
+            args.push(options.path);
+        }
+
+        return _executeCommand<ReturnTypeJson<number>>(client, args);
```

Comment:
Same here (but you can keep unchanged if you prefer)
```suggestion
        return _executeCommand(client, args);
```